python/arcticdb/adapters/azure_library_adapter.py

```diff
@@ -26,7 +28,14 @@
 @dataclass
 class ParsedQuery:
     Path_prefix: Optional[str] = None
-    CA_cert_path: str = ""
+    # winhttp is used as Azure backend support on Winodws by default; winhttp itself mainatains ca cert.
+    # The options should be left empty else libcurl will be used on Windows
+    CA_cert_path: str = (
+        ssl.get_default_verify_paths().cafile if platform.system() is not "Windows" else ""
```

Comment:
I wonder how Mac behaves? We should test it

Reply:
Mac support will be removed as CAPath is not supported by Azure C++ SDK on Mac (backed by `nss`)